### DISCUSSION TITLE
User constraints involving softbodies can never be edited or removed.

### DISCUSSION THREAD

**Discussion by jackson-waschura** [bulletphysics/bullet3#3660] [General]
There has been some fantastic work on deformable object modeling recently and I'd like to share my appreciation for that.

One issue I've run into that I'd thought I'd notify you of is that once a deformable anchor is created, it is not added to the `m_data.m_userConstraints` hashmap, and thus can never be accessed again, either to edit its properties or remove it from the simulation. This is the case for every kind of anchor involving a softbody (fixed, multibody, or rigid body).

Thanks again for your great work,
Jackson

**Comment by erwincoumans** [comment]
Indeed, deformables are preliminary/work-in-progress and we intentionally didn't expose and document the features yet until we think it is ready. Enjoy experimenting though :-)
